### PR TITLE
column and row security

### PR DESCRIPTION
**Pull request by olirice**
## What kind of change does this PR introduce?

Adds column and row security for all tables in `app` schema

### CODE REVIEW

File: supabase/migrations/20220117142137_package_tables.sql [hunk 1]
```diff
@@ -17,6 +17,11 @@ create table app.package_versions(
     semver app.semver not null,
     object_id uuid not null references storage.objects(id),
     upload_metadata jsonb, -- contents of package.json from payload
+     /*
+    yanked versions are ignored during dependency resolution
+    unless the yanked version is the only version satisfying the version requirements
+    */
+    yanked_at timestamp,
```
Comment (olirice) [inline supabase/migrations/20220117142137_package_tables.sql:24]: this change is not related to security

Comment (kiwicopple) [reply supabase/migrations/20220117142137_package_tables.sql:24]: q: what is the definition of "yanked"? When would a user yank a package?

**Review by kiwicopple — APPROVED** [review]
lgtm!

File: supabase/migrations/20220117142137_package_tables.sql (hunk 1 above)
Comment (olirice) [reply supabase/migrations/20220117142137_package_tables.sql:24]: it's similar to a soft delete

If a publisher accidentally introduced a regression in a patch release they would yank the package_version so that it would be ignored during dependency resolution unless it is the only package_version that satisfies the dependency constraint
